_posts/en/newsletters/2024-06-21-newsletter.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Bitcoin Optech Newsletter #308'
+permalink: /en/newsletters/2024/06/21/
+name: 2024-06-21-newsletter
+slug: 2024-06-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Continued discussion of PSBTs for silent payments:** several
+  developers have been discussing adding support for coordinating the
+  sending [silent payments][topic silent payments] using [PSBTs][topic
+  psbt].  Since our [previous summary][news304 sp-psbt], discussion has
+  focused on using a technique where each signer generates an _ECDH
+  share_ and a compact proof that they generated their share correctly.
+  These are added to the input section of the PSBT.  When shares from
+  all signers are received, they are combined with the receiver's silent
+  payment scan key to produce the actual key placed in the output script
+  (or multiple keys with multiple output scripts if multiple silent
+  payments are being made in the same transaction).
+
+  After all of the transaction's output scripts are known, each signer
```

Comment:
```suggestion
  After the transaction's output scripts are known, each signer
```
(minor suggestion, I think slightly better but probably okay as is, just an idea to consider)